src/main/groovy/org/shipkit/gradle/ReleaseConfiguration.java

```diff
@@ -119,6 +121,22 @@ public String getPreviousReleaseVersion() {
     public class GitHub {
 
         /**
+         * GitHub API endpoint address, for example:  https://api.github.com/
+         */
+        public String getApiUrl() {
+            return getString("gitHub.apiUrl");
+        }
+
+        /**
+         * See {@link #getApiUrl()}
+         *
+         * @param apiUrl GitHub (or gitHub enterprise) API endpoint address, for example: https://api.github.com/
+         */
```

Comment:
It would be good to resolve the endpoint URL, instead of a simple concatenation. Either by using URL class in Java (not sure if it would work in this case, I didn't test it), or a simple custom class.